docs/content/how-to/troubleshooting.md

```diff
@@ -0,0 +1,90 @@
+# Troubleshooting
+
+## How to dump the HostedCluster resources from a management cluster
+
+In order to dump the relevant HostedCluster objects we will need some prerequisites:
+
+- `cluster-admin` access to the management cluster
+- The HostedCluster `name` and the `namespace` where the CR is deployed
+- The Hypershift CLI
+
+Once we have these elements and the shell with the Kubeconfig loaded pointing to the management cluster, we will execute these commands:
+
+```bash
+CLUSTERNAME="samplecluster"
+CLUSTERNS="clusters"
+
+mkdir clusterDump-${CLUSTERNS}-${CLUSTERNAME}
+hypershift dump cluster \
+    --name ${CLUSTERNAME} \
+    --namespace ${CLUSTERNS} \
```

Comment:
We should include --dump-guest-cluster given that it's desirable most of the time.